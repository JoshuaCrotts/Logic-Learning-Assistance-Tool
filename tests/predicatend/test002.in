(s = g | ~Dga), (s = g -> As), Dga => (Ps | As)

