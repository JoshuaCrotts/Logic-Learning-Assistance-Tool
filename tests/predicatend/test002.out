1: (s = g | ~Dga)                P
2: (s = g -> As)                 P
3: Dga                           P
4: s = g                         Derived from [(s = g | ~Dga), Dga], DS
5: As                            Derived from [(s = g -> As), s = g], MP
6: (Ps | As)                     Derived from [As], ∨I
∴ (Ps | As)  ■
